### DISCUSSION TITLE
Добавить в список элементы

### DISCUSSION THREAD

**Discussion by tvideo** [totumonline/totum-ru-issues-and-discussions#372] [Как это сделать?] [answered]
Здравствуйте. Помогите с простым вопросом:
есть rowlist:
[
0:"Значение0"
1:"Значение1"
2:"Значение2"
]
каким образом его можно перебрать, что бы получить:
[
0:"Значение0"
1:"Значение0"
2:"Значение0"
3:"Значение1"
4:"Значение1"
5:"Значение1"
6:"Значение2"
7:"Значение2"
8:"Значение2"
]

Спасибо)

**Comment by erpic-ru** [comment, marked as answer]
Корявенько, но работоспособно. 

```
= : listTrain(list: $go)

     go: listReplace(list: $rowlist; action: $repeat; value: "value")
         count: listCount(list: $rowlist)
         repeat: listRepeat(item: $#value; num: $count)

~rowlist: json`[1,2,3]`
```
Можно и поизящнее придумать, конечно

**Reply by tvideo** [reply]
Спасибо
